xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -985,7 +985,7 @@ public String threadPool() {
 
   }
 
-  private static TimeUnit convertToJavaTimeUnit(org.ehcache.xml.model.TimeUnit unit) {
+  public static TimeUnit convertToJavaTimeUnit(org.ehcache.xml.model.TimeUnit unit) {
```

Comment:
Just to get to that method ... let's move it to another class instead of making ConfigurationParser public!